### ISSUE TITLE
Friendly reminder: Missing accessibility features

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#17]
Hi team,

I just wanted to bring up that it seems like some accessibility features might be missing from the project. It would be great to make the app more accessible to everyone. Thanks for considering this!

Keep up the great work!
